Dockerfile

```diff
@@ -20,24 +20,26 @@ RUN apt-get update -q && \
         git \
         curl \
         wget \
-        curl \
         software-properties-common \
     && \
     locale-gen en_US.UTF-8 && export LANG=en_US.UTF-8 && \
     add-apt-repository ppa:git-core/ppa -y && \
     add-apt-repository ppa:ondrej/php -y && \
-    echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | sudo tee /etc/apt/sources.list.d/newrelic.list && \
-    wget -O- https://download.newrelic.com/548C16BF.gpg | sudo apt-key add - && \
+    echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | tee /etc/apt/sources.list.d/newrelic.list && \
+    wget -O- https://download.newrelic.com/548C16BF.gpg | apt-key add - && \
     # Prevent newrelic install from prompting for input \
     echo newrelic-php5 newrelic-php5/application-name string "REPLACE_NEWRELIC_APP" | debconf-set-selections && \
     echo newrelic-php5 newrelic-php5/license-key string "REPLACE_NEWRELIC_LICENSE" | debconf-set-selections && \
+    # Perform cleanup \
     apt-get remove --purge -yq \
+        patch \
         software-properties-common \
+        wget \
     && \
-    rm -rf /var/lib/apt/lists/*
-
-# Ensure cleanup script is available for the next command
-ADD ./container/root/clean.sh /clean.sh
```

Comment:
lots that could be autoremoved at this point that were only being pulled out in final layer, remained in current layer